firestore-send-email/functions/__tests__/emulator/emulator-params.env

```diff
@@ -2,6 +2,10 @@ LOCATION=europe-west2
 TEMPLATES_COLLECTION=templates
 MAIL_COLLECTION=mail
 SMTP_CONNECTION_URI=smtps://fakeemail@gmail.com:secret-password@smtp.gmail.com:465
+SMTP_SERVER_HOST_AND_PORT=smtp.gmail.com:465
+SMTP_SERVER_SSL=true
```

Comment:
We may need additional `e2e` to ensure that each scenario for the configuration works.

This may help https://github.com/firebase/extensions/pull/786/files, specifically `firebase ext:dev:emulators:exec `. Is it possible to run  test scenarios where there is no username / password

Reply:
Yes, this scenario also worked (without login and pass). I have only tested it manually in local `node` because I could not test it online. Port 25 is blocked to use for Firebase Functions as I understand and I needed it to send email without credentials by Gmail. 

How do you see testing different configurations as you can only use one .env file for tests running? We could overwrite them in each test, but I am not sure if this is what are you after.